Inventory.cs

```diff
@@ -7,6 +7,12 @@ namespace InventoryManagement
     class Inventory
     {
         List<Product> products = new List<Product>();
-       
+        public void view()
```

Comment:
Methods in C# are PascalCase, also let's add an extra space before the method's declaration

```suggestion
        
        public void View()
```